dotcom-rendering/src/model/article-schema.json

```diff
@@ -1253,15 +1256,13 @@
                 "_type",
                 "activeFrom",
                 "calloutsUrl",
-                "description",
```

Comment:
When the description is an empty string, is it not going to pass the schema required rule? The current implementation that we have in frontend, would never return `None` (undefined) for any of these fields